release/player.sh

```diff
@@ -0,0 +1 @@
+../build/player.sh
```

Comment:
Building symlink leftover?

Reply:
Omg, I was wondering why this kept mysteriously dissapearing.. Will remove :)